packages/react/src/components/ImageCard/ImageCard.test.jsx

```diff
@@ -42,12 +46,6 @@ describe('ImageCard', () => {
     expect(screen.getByTestId(`${testId}-empty`)).toBeDefined();
   });
 
-  it("should show 'size not supported' when an invalid size is given", () => {
```

Comment:
How come we remove this test?

Reply:
We changed the behavior on the ImageCards when we added all the prop-type warning when using an invalid size. The preferred method was to attempt to render the content of the card as best we can, but throw a prop-type warning for using an invalid size. So, we removed the test because this message will never be shown anymore and replaced it with a test checking that the prop-type warning is shown.